### DISCUSSION TITLE
Git is tracking my system local files and showing them as untracked

### DISCUSSION THREAD

**Discussion by ikunal-04** [Cyfrin/foundry-full-course-cu#724] [Q&A] [answered]
When I ran the ```git commit -m 'First Commit!'``` all my project specified files commit successfully:

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/112847288/749b14de-77cc-4ec9-b972-7010d91cba5a)

But when I run ```git status``` my system files keeps coming in untracked files section:

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/112847288/f778e50d-528e-4098-9a2b-0fedfbc40ec1)

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/112847288/94ce8d5c-36ec-422d-b43a-2c1ff7e59f5c)

**Comment by theirrationalone** [comment]
Please set your origin. First then add data & make commits. 

To add origin:

```terminal
git remote add origin <your_repo_>
git remote -v // this will show you your fetch / push origin. 
```

**Comment by ikunal-04** [comment, marked as answer]
Actually the problem was I haven't intialised the git in my current repo i.e. why it was showing all my local files along with the one which I wanted to upload.
